README.md

```diff
@@ -68,6 +61,12 @@ SCORE_POSTGRESQL_DB_ENDPOINT = 'psl-score-db.cmpwyouptct1.us-east-2.rds.amazonaw
 SCORE_POSTGRESQL_DB_PORT = 5432
 ```
 
+Verify the installation by running the unit test suite. There are no expected test failures.
```

Comment:
For score-db, I think it would be better if we don't immediately suggest to run the test cases because this is actually inputting data into the database. Most users do not want junk data in their database. Honestly, I don't want the junk in our database but its a necessary evil since we can't easily have a test database. The tests should only be run by developers who are aware that they are inputting data. We can mention running the tests in a separate appendix if we want to reference it somewhere.

Reply:
Great point, I didn't consider that. I deleted that bit.